test/end-to-end/db/seedViper.spec.js

```diff
@@ -0,0 +1,51 @@
+import { seedDbWithViperData } from '../../../db/seedUtils';
+import db from '../../util/db';
+
+describe.only('#seedUtils', () => {
+  describe('.seedDbWithViperData', () => {
+    beforeEach(() => db.raw('DELETE FROM viper'));
```

Comment:
We should put some protection in here to ensure this never runs in STAGE or PROD